waterbox/readme.txt

```diff
@@ -9,25 +9,27 @@ It consists of a modified musl libc, and build scripts to tie it all together.
 	PREPARE A WIN10 WORKSTATION:
 	Using the guidance at (https://docs.microsoft.com/en-us/windows/wsl/wsl2-kernel & https://docs.microsoft.com/en-us/windows/wsl/install-win10):
 	1. Install WSL2 
-	2. Install Ubuntu 20.04 LTS (https://www.microsoft.com/en-us/p/ubuntu-2004-lts/9n6svws3rx71)
-	3. Clone the bizhawk repository. You can use it through /mnt or /home if you really like 
-	4. Install build tools: sudo apt-get update && sudo apt-get install gcc g++ make cmake llvm zstd
-	4b. (Note for future work: ideally the llvm installed above would not be required)
+	2. Install Ubuntu 22.04.1 LTS (https://apps.microsoft.com/store/detail/ubuntu-22041-lts/9PN20MSR04DW)

```

Comment:
Small nitpick: the LTS is now on `22.04.2` instead of `22.04.1`. The link already redirects, but might as well change the text + link accordingly.